plugin/pkg/auth/authenticator/token/webhook/webhook.go

```diff
@@ -64,11 +65,15 @@ func (w *WebhookTokenAuthenticator) AuthenticateToken(token string) (user.Info,
 		if err := result.Error(); err != nil {
 			return nil, false, err
 		}
+		var statusCode int
+		if result.StatusCode(&statusCode); statusCode < 200 || statusCode >= 300 {
```

Comment:
Didn't you want to cache 400 responses for a shorter period, or am I misremembering?